scripts/maintainer.requirements.txt

```diff
@@ -0,0 +1,6 @@
+# Python packages that are only useful to Mbed TLS maintainers.
+
+-r ci.requirements.txt
+
+# For source code analyses
+clang
```

Comment:
I wasn't aware. Out of curiosity, where do we use it? (`grep` didn't help me here)

Reply:
In some of my work in progress: https://github.com/ARMmbed/mbedtls/pull/4800, https://github.com/ARMmbed/mbedtls/pull/3952, https://github.com/ARMmbed/mbedtls/pull/5234